cmd/kubeadm/app/util/certs/util.go

```diff
@@ -222,6 +223,10 @@ func GetSparseCertTestCases(t *testing.T) []CertTestCase {
 	}
 }
 
+func CertOrKeyNameToSecretName(certOrKeyName string) string {
```

Comment:
This can be in the new `certscopy` package

Reply:
Agreed.